test/server.cpp

```diff
@@ -593,7 +595,7 @@ static int evHandler(struct mg_connection* conn, enum mg_event ev) {
 void runServer(struct mg_server* server) {
     {
         std::lock_guard<std::mutex> server_lock(server_mutex);
-        mg_set_option(server, "listening_port", SERVER_PORT);
+        mg_set_option(server, "listening_port", server_port.c_str());
```

Comment:
Why do not use the `SSL_SERVER_PORT` here?

Reply:
I think we need support HTTP and HTTPS tests at same time?

Reply:
I mean `SERVER_PORT`.

Reply:
I'm not sure whether the `gtest` will run the HTTP and HTTPS tests at same time, in my opinion, the server port should be a random number. Anyway, if you want to align the tests code, I could rollback it.

Reply:
Should be fine.
I refactored the complete http server anyway in #364 .